federation/pkg/federation-controller/util/meta.go

```diff
@@ -92,3 +97,16 @@ func DeepCopyApiTypeOrPanic(item interface{}) interface{} {
 	}
 	return result
 }
+
+func IsFederationOnlyObject(object runtime.Object) bool {
+	accessor, err := meta.Accessor(object)
+	if err != nil {
+		return false
+	}
+	annotations := accessor.GetAnnotations()
+	federationOnlyObject, exist := annotations[FederationOnlyAnnotation]
+	if exist && federationOnlyObject == "true" {
```

Comment:
(No action requried) This is a case of manufacturing a boolean where the result of the expression would suffice.  Consider the following instead:

```golang
return exist && federationOnlyObject == "true"
```

``"true"`` seems like something that should be a constant, but a quick search of the repo says otherwise.